test/envoye2e/stackdriver_plugin/fake_stackdriver.go

```diff
@@ -484,6 +502,7 @@ func RunFakeStackdriver(port uint16) error {
 	}
 	http.HandleFunc("/timeseries", fsdms.GetTimeSeries)
 	http.HandleFunc("/logentries", fsdls.GetLogEntries)
+	http.HandleFunc("/auditlogentries", fsdls.GetAuditLogEntries)
```

Comment:
why not add the log name in log entry? These are just server entries.. I think we would want to add client side logs too in future and then we would have to add more rest url's for that?

Reply:
Done.